packages/api-v2/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  rules: {
+    "monorepo-cop/no-relative-import-outside-package": "off",
```

Comment:
I was also working with Aryan on this portion. Importing does work for the backend, but the way NestJS uses it's pipes doesn't allow us to import from common. If we were to import other objects/types for use throughout the backend it would work, it only won't work if we are using the imported type for NestJS pipes, which we do when we import the DTO's.

Reply:
unfortunately building our DTOs using babel breaks class-validator validation pipes in the backend :( 

We're dealing with a ton of babel issues with building common so perhaps moving towards delegating the build step to the apps(frontend/backend) might be the move but I didn't want any of this to block them so we shall fix this in a separate ticket. Just made one https://trello.com/c/jHKm85F0/449-fix-dto-build-in-common

Reply:
sounds good!